torch_geometric/nn/conv/message_passing.py

```diff
@@ -130,6 +129,9 @@ def __init__(self, aggr: Optional[Union[str, List[str]]] = "add",
 
         # Support for GNNExplainer.
         self._explain = False
+        self.message_scale = None
+        self.message_replacement = None
+        self._gmask_explain = False
```

Comment:
Not sure why we need these arguments here. IMO, the `GraphMask` module should set these attributes (and we solely rely on `_explain` to call `self.explain_message` or not).